news/2018-07-13-cwc-2018-recap-semi-finals-finals-week-1.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Quarter Finals Recap"
+date: 2018-06-30 01:00:00 +0000
+---
+
+A double-header of news and reviews from the osu!catch World Cup 2018 this week – which three teams clinched their crucial places in Finals Week 2, and when is it all coming to a head? All the information you need is right here!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+_"Who is going to win the osu!catch World Cup 2018?"_
+
+This is a question I, [JBHyperion](https://osu.ppy.sh/users/4879508), have been asking myself for over two months now, since the Registration phase first began at the start of May. Since then, we've seen just about everything imaginable, and some sights extending beyond even that. Naturally, we are only marginally closer to knowing the answer we so desperately seek.
+
+However, over the past two weeks, our remaining twelve teams have been steadily whittled down to just three. We'll get into that a bit more later, but first, let's review the Semi Final and Finals Week 1 stages to set the scene. As always, videos of all matches are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive), so head there first if you don’t want to be spoiled!
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v279355409" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>
+
+## Semi Finals
+
+by [Dohland](https://osu.ppy.sh/users/5220511)
+
+China wasted no time advancing in the opening match, getting right down to business and delivering a swift knockout punch to the Netherlands, winning with a score of 6-1. China were looking determined to make the run from the Losers' Bracket to Grand Finals once again.
```

Comment:
`China were` -> `China was`

All countries are singular, including United States. Should use `was` or `is`. Fix other occurrences.

Reply:
China here (and similar) is used directly in reference to "the team of 6 players from China" (plural) rather than "the country of China as a whole (singular).

I've used this format consistently throughout previous articles and believe it reads the best here, technicality or otherwise.